src/Microsoft.TestPlatform.ObjectModel/Utilities/XmlRunSettingsUtilities.cs

```diff
@@ -26,6 +26,29 @@ public static ObjectModel.Architecture OSArchitecture
         {
             get
             {
+#if !NETSTANDARD1_4
+                try
+                {
+                    // use the new IsWow64Process2 to detect ARM64 as well.
+                    var processHandle = System.Diagnostics.Process.GetCurrentProcess().Handle;
+                    if (IsWow64Process2(processHandle, out ImageFileMachine pProcessMachinem, out ImageFileMachine pNativeMachine))
```

Comment:
Is the `m` at the end of `pProcessMachinem` a typo?

Reply:
I submitted this PR almost a year and a half ago, so honestly I have no clue. To be honest, I don't even remember submitting this PR.
Feel free to take what I have and work with it, but with such a delay in response and in the mean time I found other means for testing on ARM64, I don't see myself picking this PR up.

Reply:
I will most likely pick this up, when we get ARM64 hardware.